src/common/src/constants.rs

```diff
@@ -229,6 +237,11 @@ lazy_static! {
         };
         format!("{}://{}", scheme, *PUB_URL)
     };
+    pub static ref ADDITIONAL_ALLOWED_ORIGIN_SCHEMES: Vec<String> = env::var("ADDITIONAL_ALLOWED_ORIGIN_SCHEMES")
+        .unwrap_or_else(|_| String::from(""))
+        .split(' ')
+        .map(|scheme| scheme.to_string())
```

Comment:
Not sure if we need a `filter_map()` here and check for empty `String`s, or if the `.split()` filters these out already, even when we would have whitespace at the front or end. Another solution to make sure this cannot happen would be to do a `trim()` before the `split()`.

To be 100% sure to never allow any whitespace would be to add a very simply unit test.

I am just not sure if it could be possible with an input like for instance ` tauri something  `.

Reply:
Split seems to return empty values when extra spaces are used; a filter_map with a test on empty values would also get rid of unwanted extra spaces in between values.